src/utils.ts

```diff
@@ -28,6 +29,7 @@ export function getBooleanEnv(key: string): boolean {
 }
 
 export function formatAddress(address: Address, lang: string): string {
+  if (!address) return '-';
```

Comment:
This is unnecessary, as it's already ruled out by the `formatAddresses` function